doc/source/kaev_niederlausitz.md

```diff
@@ -0,0 +1,55 @@
+# Kommunaler Abfallentsorgungsverband Niederlausitz
+
+Support for schedules provided by [kaev.de](https://www.kaev.de/).
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: bsr_de
+      args:
+        abf_strasse: STRASSE
```

Comment:
I would change the order to Ort -> Ortsteil -> Strasse (here and in all other places in the document).

Reply:
changed